spec/whitehall/unpublish_document_spec.rb

```diff
@@ -0,0 +1,34 @@
+feature "Unpublishing a document by consolidating into another page on Whitehall", new: true, whitehall: true, government_frontend: true do
+  include WhitehallHelpers
+
+  let(:title) { "Unpublishing Whitehall #{SecureRandom.uuid}" }
+  let(:redirection_destination) { "http://www.dev.gov.uk/help" }
```

Comment:
Probably better to use plek to generate this url since it's validity is dependent on the env it's running within.

Reply:
Good shout. 👍